### PR TITLE
Fix/sergeyro/cleanup dependencies

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4875728227

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,128 +49,83 @@
...
-    "babel-plugin-react-docgen": "^4.2.0",
-    "babel-plugin-react-require": "^3.1.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
+    "babel-plugin-react-require": "^4.0.1",
     "boxt": "^1.1.1",
     "camelcase": "^7.0.1",
-    "chalk": "^4.1.2",
+    "chalk": "^5.3.0",
```
Comment (sahariko) [inline package.json:93]: Where are we using chalk?

File: package.json [hunk 2]
```diff
@@ -49,128 +49,83 @@
...
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-storybook": "^0.6.12",
-    "execa": "^5.1.1",
-    "file-loader": "^6.2.0",
+    "eslint-plugin-storybook": "^0.6.13",
+    "execa": "^7.2.0",
```
Comment (sahariko) [inline package.json:102]: Where are we using execa?

File: package.json [hunk 3]
```diff
@@ -49,128 +49,83 @@
...
-    "jest": "^27.5.1",
-    "jest-environment-jsdom": "^27.5.1",
-    "jest-pnp-resolver": "^1.2.3",
-    "jest-watch-typeahead": "^1.1.0",
+    "jest": "^29.6.1",
+    "jest-environment-jsdom": "^29.6.1",
     "lerna-changelog": "^2.2.0",
     "monday-ui-style": "^0.1.172",
     "postcss": "^8.4.25",
     "postcss-scss": "^4.0.6",
```
Comment (sahariko) [inline package.json:110]: Are we using this?

File: package.json [hunk 4]
```diff
@@ -49,128 +49,83 @@
...
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
     "process": "^0.11.10",
```
Comment (sahariko) [inline package.json:112]: What is this?

File: package.json [hunk 5]
```diff
@@ -49,128 +49,83 @@
...
     "prop-types": "^15.8.1",
```
Comment (sahariko) [inline package.json:113]: Are we using this?

File: package.json [hunk 6]
```diff
@@ -49,128 +49,83 @@
...
-    "react-transition-group": "^4.4.5",
-    "regenerator-runtime": "^0.13.11",
-    "remark-gfm": "^3.0.1",
-    "resolve-url-loader": "^5.0.0",
     "rollup": "^3.26.2",
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "rollup-plugin-postcss": "^4.0.2",
     "rollup-plugin-typescript2": "^0.35.0",
     "sass": "^1.63.6",
     "sass-loader": "^13.3.2",
```
Comment (sahariko) [inline package.json:123]: Is this in use?

File: package.json (hunk 1 above)
Comment (SergeyRoyt) [reply package.json:93]: Both chalk and execa are used in release and prerelease scripts, which I copied from monday-ui-react-core

File: package.json (hunk 2 above)
Comment (SergeyRoyt) [reply package.json:102]: Both chalk and execa are used in release and prerelease scripts, which I copied from monday-ui-react-core

File: package.json (hunk 3 above)
Comment (SergeyRoyt) [reply package.json:110]: Not yet, but probably later we'll add some tests, so I thought it's better to keep them

File: package.json (hunk 6 above)
Comment (SergeyRoyt) [reply package.json:123]: Not sure, I thought it might be used by storybook - to load styles into the stories

File: package.json (hunk 5 above)
Comment (SergeyRoyt) [reply package.json:113]: Not yet, but maybe in the future... as we probably don't want to convert this components to TS, but do want to export props types outside

File: package.json (hunk 4 above)
Comment (SergeyRoyt) [reply package.json:112]: Probably can be deleted

File: package.json (hunk 6 above)
Comment (sahariko) [reply package.json:123]: This is a Webpack-related dependency, "loader" is a Webpack terminology.

**Pull request closed without merging**